appcoins-billing/src/main/java/com/appcoins/sdk/billing/GetSkuDetailsService.java

```diff
@@ -0,0 +1,65 @@
+package com.appcoins.sdk.billing;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.util.List;
+
+public class GetSkuDetailsService {
+
+  private final String serviceUrl;
+
+  public GetSkuDetailsService(final String serviceUrl) {
+    this.serviceUrl = serviceUrl;
+  }
+
+  public String getSkuDetailsForPackageName(final String packageName, final List <String> sku) {
+    String response = "";
+    URL url = null;
+    try {
+      url = new URL(buildURL(packageName,sku));
+
+      HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+      connection.setRequestMethod("GET");
+      connection.connect();
+      BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+      String inputLine;
+
+      while ((inputLine = in.readLine()) != null) {
+        response += inputLine;
+      }
+
+      if (in != null) {
+        in.close();
+      }
+
+      if (connection != null) {
+        connection.disconnect();
+      }
+    } catch (MalformedURLException e) {
+      e.printStackTrace();
+      response = "";
+    } catch (ProtocolException e) {
+      e.printStackTrace();
+      response = "";
+    } catch (IOException e) {
+      e.printStackTrace();
+      response = "";
+    }
+
+    return response;
+  }
+
+  private String buildURL(String packageName, List<String> sku) {
+    String url = serviceUrl + "/inapp/8.20180518/packages/packageName/products?names=".replaceFirst(
```

Comment:
the hardcoded path should be a final at the top of the file for better maintainability of the code. If someone else would need to change the path they wouldn't have to fish that var in the middle of the file.